packages/app/src/files-manager/helpers/display-space.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import round from "lodash/round"
+
+export const displaySpace = (mebibytesSize: number): string => {
```

Comment:
🟢
The function name no means what it does. This function converts one unit to another (view unit). From the view perspective, maybe display space but It does not matter how parents use this function.

Reply:
Name changed to `convertFromMebibytes`. [5d0cd24](https://github.com/mudita/mudita-center/pull/598/commits/5d0cd24e83476e4283bfc8a768ebb6b9a844767c)